### PR TITLE
update properties

### PR DESCRIPTION
**Pull request by VladSpitsin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/932
* change max length of first name, last name, middle name

### CODE REVIEW

File: src/UserService.Validation/User/CreateUserRequestValidator.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>,
             .WithMessage("Middle name must not contain numbers.")
             .Must(x => !SpecialCharactersRegex.IsMatch(x))
             .WithMessage("Middle name must not contain special characters.")
-            .MaximumLength(32)
+            .MaximumLength(45)
             .WithMessage("Middle name is too long.")
```
Comment (Cheephot) [inline src/UserService.Validation/User/CreateUserRequestValidator.cs:54]: move to 53 line

**Review by Cheephot — APPROVED** [review]

File: src/UserService.Validation/User/CreateUserRequestValidator.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/UserService.Validation/User/CreateUserRequestValidator.cs:54]: why? here every message comes after verification

**Review by BelehovEgor — APPROVED** [review]

File: src/UserService.Validation/User/EditUserRequestValidator.cs [hunk 2]
```diff
@@ -78,7 +73,7 @@ private void HandleInternalPropertyValidation(Operation<EditUserRequest> request
         {
           { x => !string.IsNullOrEmpty(x.value?.ToString().Trim()), "Last name must not be empty." },
           { x => !NameRegex.IsMatch(x.value.ToString()), "Last name must not contain numbers." },
-          { x => x.value.ToString().Trim().Length < 32, "Last name is too long." },
+          { x => x.value.ToString().Trim().Length < 45, "Last name is too long." },
```
Comment (MariyaSin) [inline src/UserService.Validation/User/EditUserRequestValidator.cs:76]: change to 46, you must include 45

File: src/UserService.Validation/User/EditUserRequestValidator.cs [hunk 3]
```diff
@@ -64,7 +59,7 @@ private void HandleInternalPropertyValidation(Operation<EditUserRequest> request
         {
           { x => !string.IsNullOrEmpty(x.value?.ToString().Trim()), "First name must not be empty." },
           { x => !NameRegex.IsMatch(x.value.ToString()), "First name must not contain numbers." },
-          { x => x.value.ToString().Trim().Length < 32, "First name is too long." },
+          { x => x.value.ToString().Trim().Length < 45, "First name is too long." },
```
Comment (MariyaSin) [inline src/UserService.Validation/User/EditUserRequestValidator.cs:62]: change to 46, you must include 45

File: src/UserService.Validation/User/EditUserRequestValidator.cs (hunk 2 above)
Comment (VladSpitsin) [reply src/UserService.Validation/User/EditUserRequestValidator.cs]: changed

File: src/UserService.Validation/User/EditUserRequestValidator.cs (hunk 3 above)
Comment (VladSpitsin) [reply src/UserService.Validation/User/EditUserRequestValidator.cs]: changed

File: src/UserService.Validation/User/CreateUserRequestValidator.cs (hunk 1 above)
Comment (Cheephot) [reply src/UserService.Validation/User/CreateUserRequestValidator.cs]: I mean: .MaximumLength(45).WithMessage("Middle name is too long."). @DarkFelas spoke, what so better

Comment (VladSpitsin) [reply src/UserService.Validation/User/CreateUserRequestValidator.cs]: done

**Pull request closed without merging**